contracts/Core/AssetManager.sol

```diff
@@ -102,14 +118,87 @@ contract AssetManager is ACL, AssetStorage {
             url, 
             selector, 
             name, 
-            repeat, 
-            msg.sender, 
-            msg.value, 
+            repeat,  
             block.timestamp, 
             assetTypes.Job
         );
     }
 
+    function updateJob(
+        uint256 jobID,
+        string calldata url,
+        string calldata selector
+    ) 
+        external 
+        onlyRole(parameters.getAssetCreatorHash()) 
+    {
+        require(jobs[jobID].assetType==uint256(assetTypes.Job),"Job ID not present");
+        require(jobs[jobID].active, "Job has been inactivated");
+
+        uint256 epoch = parameters.getEpoch();
+
+        jobs[jobID].url = url;
+        jobs[jobID].selector = selector;
+
+        emit JobUpdated(
+        jobID,
+        epoch,
+        url,
+        selector,
+        block.timestamp
+        );
+
+    }
+
+    function updateAssetStatus(
+        uint256 id,
+        uint256 activationMethod
```

Comment:
Can we have bool as type directly here, and can we have variable name like isActive ?

Reply:
Alright, bool can be used here also
Do you want isActive to replace activationMethod or the active variable in the job and collection structs?

Reply:
Hmm, my suggestion was for activationMethod only, active in structs should be fine IMO